### PR TITLE
Add layout, homepage

### PR DESCRIPTION
**Pull request by SirajChokshi**
# Added homepage components, layout

## Changes

- Components
    - Filtering (SearchInput, MultiSelect, Tags)
    - OrgCard
    - Feed
    - Layout
    - Navigation

### CODE REVIEW

File: app/components/Feed.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      // filter by query
+      ?.filter(
+        (res) =>
+          !filters.query ||
+          (res?.name + res?.desc + res?.categories?.join('-'))
+            ?.toLowerCase()
+            .toLowerCase()
+            .includes(filters.query.toLowerCase()),
+      );
+    // filter by founding date
```
Comment (aaronalberg) [inline app/components/Feed.js:73]: should this still be here

File: app/components/Feed.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styles from '../styles/Feed.module.css';
+
+import OrgCard from '../components/OrgCard';
+
+function hasSharedElem(a, b) {
```
Comment (aaronalberg) [inline app/components/Feed.js:15]: docstring?

File: app/components/Feed.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  for (const elemB of b) {
+    if (a.some((elemA) => elemA === elemB)) {
```
Comment (aaronalberg) [inline app/components/Feed.js:17]: and I don't 100% know what this is specifically doing

File: app/components/Feed.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+          !filters.query ||
+          (res?.name + res?.desc + res?.categories?.join('-'))
+            ?.toLowerCase()
+            .toLowerCase()
+            .includes(filters.query.toLowerCase()),
+      );
+
+    // TODO: Add dates to models and enable filtering
+    // filter by founding date
+    // ?.filter(
```
Comment (daviskeene) [inline app/components/Feed.js:74]: might want to take this out? leave the TODO but maybe scrap the implementation

File: app/components/Layout.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+export default function Layout({ title, children }) {
+  return (
+    <>
+      <Nav />
+      <div className={styles.container}>
+        <Head>
+          <title>
+            {title ? `${title} | Startup Directory` : 'Startup Directory'}
+          </title>
+          <link rel="icon" href="/favicon.ico" />
```
Comment (daviskeene) [inline app/components/Layout.js:21]: Should change the favicon to the founders logo for the future

File: app/components/Tags.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+ *  fadeRight: boolean,
+ *  onClick: (e: Event) => void
+ * }} props
+ * @returns {React.Component}
+ */
+const Tags = React.forwardRef(
+  ({ tags, onClick, style, tagStyle, fadeRight }, ref) => {
+    return (
+      <ul
+        style={{
```
Comment (daviskeene) [inline app/components/Tags.js:19]: lots of inline css in this file - consider moving to a separate file? like `'../styles/Tags.module.css'`

**Review by daviskeene — APPROVED** [review]
This is great to merge as is; left a few small comments.

File: app/components/Layout.js (hunk 5 above)
Comment (SirajChokshi) [reply app/components/Layout.js:21]: Agreed, but unrelated to this PR

File: app/components/Tags.js (hunk 6 above)
Comment (SirajChokshi) [reply app/components/Tags.js:19]: I think it makes more sense here since we want our prop overrides to be in-scope

**Pull request merged**